java/terrain-exaggeration/src/main/java/com/esri/arcgisruntime/sample/terrainexaggeration/MainActivity.java

```diff
@@ -72,7 +73,7 @@ protected void onCreate(Bundle savedInstanceState) {
       }
 
       @Override public void onStopTrackingTouch(SeekBar seekBar) {
-
+        seekBar.setProgress(Math.max(1, seekBar.getProgress()));
```

Comment:
maybe just add a comment like 

`// disallow any progress value below 1`

the only reason I suggest it is because I had to look up what Math.max(int, int) does :P